src/Domain/Insights/ForbiddenBadClasses.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+declare(strict_types=1);
+
+namespace NunoMaduro\PhpInsights\Domain\Insights;
+
+use NunoMaduro\PhpInsights\Domain\Contracts\HasDetails;
+use NunoMaduro\PhpInsights\Domain\Details;
+
+final class ForbiddenBadClasses extends Insight implements HasDetails
+{
+    public function hasIssue(): bool
+    {
+        return count($this->getDetails()) > 0;
+    }
+
+    public function getTitle(): string
+    {
+        return (string) ($this->config['title'] ?? 'Bad classes are forbidden.');
```

Comment:
Beyond being forbidden, they are unusable, right?

I may reword this Class just BadClasses, with `Unusable class` or something like that :upside_down_face: